packages/waku/src/router/base-types.ts

```diff
@@ -19,3 +19,25 @@ export interface CreatePagesConfig {
 /** Props for pages when using `createPages` */
 export type PageProps<Path extends PagePath<CreatePagesConfig>> =
   PropsForPages<Path>;
+
+type StaticConfig = {
```

Comment:
Once again, avoid adding createPage things in base-types.ts. This file is basically for defineRouter with a few exceptions.

We can define somewhere else and re-export it from here so that it allows importing from `waku/router`.

Reply:
This is just for example for now. I will clean everything up when we decide on the best strategy for solving the `getConfig` problem.